demo/guides/request-review-takeover.html

```
@@ -0,0 +1,117 @@
+              </div>
+              <div class="field-card">
+                <h3>Mutation gate</h3>
+                <p>Reject missing mutating headers, stale WorkSession revision, previous event hash mismatch, actor/body mismatch, and unauthorized Actor.</p>
+              </div>
+            </div>
+          </article>
+
+          <article class="spec-panel">
+            <h2>Lifecycle</h2>
+            <pre class="code-sample"><code>PolicyDecision denies or flags action
+  -> Request created with blocking_scope
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)